features/pagination.feature

```diff
@@ -10,6 +10,43 @@ Feature: Use Pagination to filter result
     | 2  | Banana                | fruit,yellow | 2001-07-21 | 2003-08-30   | true      | 0.50$ |
     | 3  | Sausages              | meat         | 2010-04-11 | 2005-02-01   | true      | 5.30$ |
 
-  Scenario: Return all the data set when no parameters are given
-    When I request the url "/dataset"
+  Scenario: Return all the data when no parameters are given
```

Comment:
@mkoert This is the file that needs to be looked for to determine if i'm on the right track